Kickstarter-iOS/Assets.xcassets/icons/credit cards/icon--apple_pay.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "icon--apple_pay.pdf"
```

Comment:
I think we've typically preferred dashes over underscores for asset names, so this should be `icon--apple-pay` instead of `icon--apple_pay`